parser_library/src/lib_config.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+#include "lib_config.h"
+
+static std::shared_ptr<lib_config> config_instance = std::make_shared<lib_config>();
```

Comment:
is static needed here?

Reply:
could be anonymous namespace instead...

But I am still wondering if this should be part of the server's state.